src/test/java/com/amazon/pvar/tspoc/merlin/CallGraphSerializationTests.java

```diff
@@ -0,0 +1,43 @@
+package com.amazon.pvar.tspoc.merlin;
+
+import com.amazon.pvar.tspoc.merlin.ir.FlowgraphUtils;
+import com.amazon.pvar.tspoc.merlin.solver.CallGraph;
+import com.google.gson.Gson;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonPrimitive;
+import dk.brics.tajs.flowgraph.jsnodes.CallNode;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+
+public class CallGraphSerializationTests extends AbstractCallGraphTest {
+
+    @Test
+    public void canSerializeCallGraphsToJson() throws FileNotFoundException {
```

Comment:
As-is, this test is a bit hard to read. I prefer to split up tests into three, commented sections:
1. "Arrange" - Where you set up the test structures: mocks, expected results, etc.
2. "Act" - Where you execute the function under test
3. "Assert" - Where you do your assertion(s) 

Can we group the setup together at least?

Reply:
Sure, I'll do that.